spec/models/field_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Field, type: :model do
+subject(:field) { FactoryBot.build(:field) } 
```

Comment:
just call `build(:field)`. if it fails because of the unknown method, properly configure spec_helper

Reply:
Thanks. I've taken this into account in my last commit.